uplink/src/collector/systemstats.rs

```diff
@@ -540,6 +549,18 @@ impl StatCollector {
                 error!("Couldn't send component stats: {}", e);
             }
         }
+        let files = glob::glob("/sys/devices/virtual/thermal/thermal_zone*/temp").unwrap();
+        for thermal_zone in files {
+            let path = thermal_zone.unwrap();
```

Comment:
This will crash if process doesn't have permission to read directories.

Reply:
Have converted these to errors instead of panics